### ISSUE TITLE
Native app for Linux and macOS + Windows 7 & 8

### ISSUE THREAD

**Issue by theweavrs** [theweavrs/BreadPlayer#135]

**Comment by Bond-009** [comment]
I don't think that it's worth the effort to create a Windows 7 or 8 app. Since you support 10 and thats the latest. That said I would LOVE macOS and Linux support.

**Comment by theweavrs** [comment]
Making an app for Linux and macOS will automatically enable Windows 7,8,8.1 support because it is supposed to be portable. But you are right, users will eventually move on to Windows 10.

**Comment by ahmedpiosol** [comment]
i really doubt that even now there is people who still uses Windows XP and it have more share on desktops than linux so "users will eventually move on to Windows 10" may take a very very very long time
support for windows 7 specially is a good move

**Comment by applesea2** [comment]
Remember that this is a UWP app. That means it works almost entirely and only on platforms that support UWP. Creating an app for Windows 7/Linux means rewriting the app from scratch for the two OS's.

**Comment by jaigak** [comment]
You could use [Avalonia UI](https://github.com/AvaloniaUI/Avalonia) for building a cross-platform .NET app. It has Fluent Design for a modern design.
